ui/node/identifiers.go

```diff
@@ -27,4 +27,12 @@ const (
 	WebAuthnRegisterDisplayName = "webauthn_register_displayname"
 	WebAuthnRemove              = "webauthn_remove"
 	WebAuthnScript              = "webauthn_script"
+	PasskeyRegisterTrigger      = "passkey_register_trigger"
+	PasskeyRegister             = "passkey_register"
+	PasskeySettingsRegister     = "passkey_settings_register"
+	PasskeyCreateData           = "create_passkey_data"
+	PasskeyLogin                = "passkey_login"
+	PasskeyChallenge            = "passkey_challenge"
+	PasskeyLoginTrigger         = "login_with_passkey"
```

Comment:
All other names are prefixed with the credential type - please keep consistency?